examples/platform/efr32/args.gni

```diff
@@ -16,6 +16,16 @@ import("//build_overrides/chip.gni")
 
 import("${chip_root}/src/platform/EFR32/args.gni")
 
+import("//build_overrides/pigweed.gni")
+
+declare_args() {
+  dir_pw_log_chip =
+      get_path_info("$chip_root/src/lib/support/pw_log_chip", "abspath")
```

Comment:
Why use an absolute path here? Generally we want paths in build files to be relative to the source root ("//"), at least until they become command lines, at which point we rebase them to be relative to the build directory.

Absolute paths are discouraged and can cause real harm.. e.g. caches misses in distributed builds.

Reply:
Done